desktop/hardened-desktop/admin-controls/configure-ac.md

```diff
@@ -0,0 +1,129 @@
+---
+description: admin controls for desktop
+keywords: admin, controls, rootless, enhanced container isolation
+title: Configure Admin Controls
+--- 
+
+>Note
+>
+>Admin Controls is currently in [Early Access](../../../release-lifecycle.md#early-access-ea) and available to Docker Business customers only. 
+
+This page contains information about how administrators can configure Admin Controls to specify and lock configuration parameters to create a standardized Docker Desktop environment across the organization.
+
+Admin Controls is designed specifically for organizations who don’t give developers root access to their machines.
+
+## Prerequisite
+
+You need to [configure a registry.json to enforce sign-in](../../../docker-hub/configure-sign-in.md). For this configuration to take effect, Docker Desktop users must authenticate to your organization. 
+
+## Step one: Place the `admin-settings.json` file in the correct location
+
+Place the `admin-settings.json` file on your developers' machines in the following locations:
+
+- Mac: /Library/Application Support/com.docker.docker/admin-settings.json
+- Windows: /ProgramData/DockerDesktop/admin-settings.json
+- Linux - /usr/share/docker-desktop/admin-settings.json 
+
+By placing this file in the above protected directories, end users are unable to modify it.
+
+>Note
+>
+> It is assumed that you have the ability to push the `admin-settings.json` settings file to the locations specified above through a device management software such as [Jamf](https://www.jamf.com/lp/en-gb/apple-mobile-device-management-mdm-jamf-shared/?attr=google_ads-brand-search-shared&gclid=CjwKCAjw1ICZBhAzEiwAFfvFhEXjayUAi8FHHv1JJitFPb47C_q_RCySTmF86twF1qJc_6GST-YDmhoCuJsQAvD_BwE).
+
+## Step two: Add the key value pairs for the admin controls you want to lock in
+
+>Note
+>
+>Some of the configuration parameters only apply to Windows. This is highlighted in the table below.
+
+The `admin-settings.json` file requires a nested list of configuration parameters, each of which must contain the  `locked` parameter. 
+If set to `true`, users are not able to edit this setting from Docker Desktop or the CLI. 
+If set to `false`, users can change this setting from Docker Desktop or the CLI by directly editing the `settings.json` file. If this setting is omitted, the default value is `false`.
```

Comment:
i think we should also mention here that admins don't have to enter every single key: value pair in the admin-settings.json for the feature to work correctly (note - i'm assuming that's the case but probably best to check with Manu !). 

e.g. if im an admin and i just want to configure proxy settings then i dont have to also enter values for 'useWindowsContainers', 'linuxVM', etc

Reply:
What happens to the windows values on macOS ? Can we call out that values that are irrelevant to your platform will be ignored ?